cmd/kubeadm/test/token_test.go

```diff
@@ -29,12 +29,20 @@ const (
 )
 
 var kubeadmPath string
+var testCmd = true
 
 func init() {
 	flag.StringVar(&kubeadmPath, "kubeadm-path", filepath.Join(os.Getenv("KUBE_ROOT"), "cluster/kubeadm.sh"), "Location of kubeadm")
+	if _, err := os.Stat(kubeadmPath); os.IsNotExist(err) {
```

Comment:
I've seen this type of check happen in `init()` blocks before, especially when trying to test subprocesses:

https://speakerdeck.com/mitchellh/advanced-testing-with-go